pkg/daemon/ceph/osd/encryption.go

```diff
@@ -275,7 +275,7 @@ func addEncryptionKey(context *clusterd.Context, disk, passphrase, newPassphrase
 		// if newPassphrase is not one in the slot, then remove the key slot and
```

Comment:
can you please update the comment as well to reflect why `newPassPhrase` is not used.

Reply:
```suggestion
		// if newPassphrase is not one in the slot, then remove the key slot using current passphrase and then
```
@black-dragon74 ^